### ISSUE TITLE
Bungee Support

### ISSUE THREAD

**Issue by Games4k** [rafedup27/AntiBot-Ultra#1]
Hello. Is it possible to make antibotultra for bungeecord?

**Comment by rafedup27** [comment]
Yes! Now I'm focused on a new stable version of antibot-ultra then i will release a bungeecord version! Sorry for my late "response" should i say! Thank you for your interest in this plugin!
